index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta charset="UTF-8">
+        <title>Hello Microverse</title>
+    </head>
```

Comment:
- [OPTIONAL] You should have the `link` tag inside your `hard` tag container in order to link your index.html file with the style.css file, Actually, there is no connection between them, the index.html file can't recognize where your CSS file is exactly located. In order to solve this, you need to add this line after line 6 inside the index.html.  this should be like that:
 ` <link rel="stylesheet" href="./styles.css">`


### This is only optional because due to oversight the previous reviewer probably missed them. However, it is important.